cmd/habitat/community/consensus/raft/helpers.go

```diff
@@ -40,3 +43,23 @@ func getCommunityAddress(communityID string) string {
 func getCommunityRaftDirectory(communityID string) string {
 	return filepath.Join(compass.CommunitiesPath(), communityID, "raft")
 }
+
```

Comment:
this is like the protocol version, correct?

Reply:
libp2p uses the protocol id to tell different nodes connected on a stream what to expect for data